setup.cfg

```
@@ -22,6 +22,7 @@ install_requires =
 	numpy
 	orjson
     h5py
+	tifffile
```

```suggestion
	h5py
	tifffile
```

Just to normalize `h5py` indentation